Doc/library/gzip.rst

```diff
@@ -234,9 +236,26 @@ Command line options
 
    If *file* is not specified, read from :attr:`sys.stdin`.
 
+.. cmdoption:: --fast
+
+   Indicates the fastest compression method (less compression).
+
+   .. code-block:: shell-session
+
+      $ python -m gzip --fast file
```

Comment:
"unversionned" python must not be used: please write python3 (and change the existing documentation for "file").

I'm not sure that it's useful to add an example of a command for each command line option.

Reply:
Actually, most examples in the current Doc *do* use an unversioned python for `-m` examples.  Only a few currently use `python3 -m`.   We should probably decide on which is better and fix them all.  But that should be a separate issue and PR.

Reply:
I think it's better to use `python`instead of `python3` since that will add the extra work of replacing all `python3` with `python4` when we have a release in future :) Many examples will be compatible with both versions and also might cause issues in backporting doc fixes between two major versions . The URL and drop-down at the top already indicates python3.X version. I might be missing a use case here.

Reply:
I remove the example with the command line and in this way, just avoid the "un.versioned" python.